### PR TITLE
Bugfix(#3466)

### PR DESCRIPTION
**Pull request by VampyrBytes**
Allows oxyloss regen (at normal rate) and turns off oxygen and toxin
hud warnings when you don't need to breathe

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Don't personally like that nobreathing heals oxygen damage--pretty much eliminates the threat of bloodloss altogether, not to mention makes individuals partially immune to heart attacks.

People can just take some salbutamol/perflu/get chucked into cryo and it'll heal the O2 damage.

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -295,14 +295,14 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 					if(istype(O)) O.add_autopsy_data("Radiation Poisoning", damage)
 
 /mob/living/carbon/human/breathe()
-	if(reagents.has_reagent("lexorin"))
+
+	if((NO_BREATH in mutations) || (species && (species.flags & NO_BREATHE)) || reagents.has_reagent("lexorin"))
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/life.dm]: I don't think we have lexorin anymore...

Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/life.dm]: we do.

Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/life.dm]: Oh? Where does that pop up?

Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/life.dm]: random cigs.

**Comment by Fox-McCloud** [comment]
Just tested this; this makes people who have no breath/lexorin recover 10 Oxygen damage every 4 cycles instead of 5.

**Pull request merged**